class/nomenclature.class.php

```diff
@@ -1015,9 +1015,9 @@ static function getDefaultNomenclature(&$PDOdb, $fk_product, $qty_ref = 0)
 		return false;
 	}
 
-    static function resetDefaultNomenclature(&$PDOdb, $fk_product)
+    static function resetDefaultNomenclature(&$PDOdb, $fk_product, $object_type)
```

Comment:
```suggestion
    static function resetDefaultNomenclature(&$PDOdb, $fk_object, $object_type = 'product')
```
To avoid fatal errors for external modules calling this method for old product behavior